.github/workflows/e2e-test.yml

```
@@ -22,15 +22,15 @@ jobs:
 
       - name: Initiate Dockware
         run: |
-          docker pull dockware/dev:6.4.20.1
+          docker pull dockware/dev:latest
```

I think it's better for the longrun to bind to a specific version
```suggestion
          docker pull dockware/dev:6.5
```